modules/home/api.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace Elementor\Modules\Home;
+
+class API {
+
+	const HOME_SCREEN_DATA_URL = 'https://assets.stg.elementor.red/home-screen/v1/home-screen.json';
```

Comment:
Is this supposed to be stg....red?

Reply:
I am waiting for this PR to be merged: https://github.com/elementor/editor-assets/pull/44. I can only update the URL to non-staging URL, once the other PR has been merged.

Reply:
Updated.